tests/flytekit/unit/core/test_local_cache.py

```diff
@@ -0,0 +1,78 @@
+from pytest import fixture
```

Comment:
can we test some more complex input and output types, like dataclasses, schemas, files?